.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,136 @@
+# Contribution Guidelines
+
+🙏 Thank you for taking the time to contribute!
+
+Your input is deeply valued, whether an issue, a pull request, or even feedback, regardless of size, content or scope.
+
+## Table of contents
+
+[👶 Getting started](#getting-started)
+
+[📔 Code of Conduct](#code-of-conduct)
+
+[🗃 License](#license)
+
+[📜 Issues](#issues)
+
+[🚩 Pull requests](#pull-requests)
+
+[💻 Coding guidelines](#coding-guidelines)
+
+[🏆 Recognition model](#recognition-model)
+
+[❓ Questions](#questions)
+
+## Getting started
+
+Please familiarize yourself with the specific [project structure][proj_readme] and the book content. Please make sure you read the "Development" section in the [README][book_readme] file.
```

Comment:
copy&paste from other repos with this line adjusted - in all other repos it refers more to a package with it's docs which is not suitable here.